lms/services/document.py

```diff
@@ -0,0 +1,39 @@
+import re
+from dataclasses import dataclass
+from typing import Optional
+
+DOCUMENT_URL_REGEXES = {
```

Comment:
All regexp are now in this file.

I'm using dictionary here because it felt wrong not to but we don't use the key.